### PR TITLE
add option which hides events which not feets time

### PR DESCRIPTION
**Pull request by AleksSem**
As you mention [in previous mr](https://github.com/Serhioromano/bootstrap-calendar/pull/637#issuecomment-235310897). I add an additional parameter.

### CODE REVIEW

File: js/calendar.js [hunk 1]
```diff
@@ -530,19 +534,36 @@ if(!String.prototype.formatNum) {
 				e.end_hour = f.getDate() + ' ' + $self.locale['ms' + f.getMonth()] + ' ' + e.end_hour;
 			}
 
-			if(e.start < start.getTime() && e.end > end.getTime()) {
-				data.all_day.push(e);
-				return;
-			}
+			if(!show_fits_time_events) {
```
Comment (Serhioromano) [inline js/calendar.js]: I think we can use var `$self.options.show_events_which_fits_time`  right here and get rid of `var show_fits_time_events = $self.options.show_events_which_fits_time;`

**Comment by Serhioromano** [comment]
Perfect! That is nice. One note though. When you apply changes try not to change formatting. I understand that indentation have to be right but that pollutes PR. Next time try to commit with PR only lines that related to your changes to logic. If you want to adjust formatting you can create new PR for that.

In this PR it is OK because not a lot of format changes are there. But sometimes it might be very difficult to analyze changes when there are a lot of diffs.

**Comment by AleksSem** [comment]
Thanks i'll keep that in mind next time.

**Comment by Serhioromano** [comment]
Do not forget to apply changes I've recommended in the other comment so I could accept this PR.

**Comment by AleksSem** [comment]
Done

**Comment by Serhioromano** [comment]
Good. Let me make the logic strait here.

    //shows events which fits between time_start and time_end

This parameter is false, then we do not show events that are between start and end time but show those that only partially between end and start?

**Comment by AleksSem** [comment]
If this parameter is false then we show all events (default logic), if true then only those which fits the day.

**Pull request merged**